src/modules/Slideshow/styles.module.scss

```diff
@@ -9,10 +9,10 @@
     width: 100%;
     height: 100vh;
     position: relative;
-    margin-top: -50px;
-    background: url("../../images/bg.jpg");
-    background-position: center;
+    background: url("../../images/bg.jpg");	
```

Comment:
this was just a move, but I do want to remove this background at some point for further optimizations.